scss/modules/_hero.scss

```diff
@@ -43,6 +43,58 @@
   width: 100%;
 }
 
+.hero--home {
+  @include u-bg--neutral();
+
+  .hero__banner {
+    background-image: url('../img/fec-table-tint.png');
+    width: 100%;
+    height: u(22rem);
+    position: absolute;
+  }
+
+  h2 {
+    color: $inverse-translucent;
```

Comment:
I _think_ this should actually just be white, so `$inverse`. I think the shading you see is an artifact of the jpeg maybe? @jenniferthibault can you verify that the main title over the image of the chairs should be full opacity white?